### PR TITLE
Add Python Load Tests Combine Dataflow Batch github action

### PR DESCRIPTION
**Pull request by Amar3tto**
The following jenkins job has been migrated:

https://ci-beam.apache.org/job/beam_LoadTests_Python_Combine_Dataflow_Batch
https://ci-beam.apache.org/job/beam_LoadTests_Python_Combine_Dataflow_Batch_PR

### How has it been tested?
The job has been tested on Self-hosted GKE runners ubuntu 20.04.
The following test cases are passed:

- by adding a comment to the PR

Screenshot from 2023-09-12 19-21-49

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: .github/workflows/beam_LoadTests_Python_Combine_Dataflow_Batch.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - name: Prepare configs
+        id: set_configs
+        shell: bash
+        run: |
+          CURDATE=$(date '+%m%d%H%M%S' --utc)
+          CURCONFIG=$(grep -v "^#.*" ./.github/workflows/load-tests-job-configs/config_Combine_Python_Batch_2GB_10b.txt | tr '\n' ' ')
+          CURCONFIG=$(echo "${CURCONFIG/load-tests-python-dataflow-batch-combine-1-/load-tests-python-dataflow-batch-combine-1-$CURDATE}")
```
Comment (damccorm) [inline .github/workflows/beam_LoadTests_Python_Combine_Dataflow_Batch.yml:82]: What is this 2nd echo doing (here and in other configs)?

Comment (Amar3tto) [reply .github/workflows/beam_LoadTests_Python_Combine_Dataflow_Batch.yml:82]: It replaces `job_name` substring with the "name+current date".
In .groovy it was: 
`def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))`
...
`job_name             : "load-tests-python-dataflow-batch-combine-1-${now}"`

Comment (damccorm) [reply .github/workflows/beam_LoadTests_Python_Combine_Dataflow_Batch.yml:82]: Could you add an explanatory comment to this section explaining what we are doing? It is hard to gather intent from the code

Comment (Amar3tto) [reply .github/workflows/beam_LoadTests_Python_Combine_Dataflow_Batch.yml]: Done

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**